azure-application-insights-spring-boot-starter/README.md

```diff
@@ -133,16 +133,16 @@ azure.application-insights.logger.level=error
 azure.application-insights.channel.in-process.developer-mode=false
 # Endpoint address, Default value: https://dc.services.visualstudio.com/v2/track.
 azure.application-insights.channel.in-process.endpoint-address=https://dc.services.visualstudio.com/v2/track
-# Maximum count of telemetries that will be batched before sending. Default value: 500.
+# Maximum count of telemetries that will be batched before sending. Must be between 1 and 1000. Default value: 500.
 azure.application-insights.channel.in-process.max-telemetry-buffer-capacity=500
-# Interval to send telemetry. Default value: 5 seconds.
+# Interval to send telemetry. Must be between 1 and 300. Default value: 5 seconds.
 azure.application-insights.channel.in-process.flush-interval-in-seconds=5
-# Size of disk that we can use. Default value: 10 megabytes.
+# Size of disk that we can use. Must be between 1 and 1000. Default value: 10 megabytes.
```

Comment:
Does't the default and maximum value both seem to be too low. What do you think? Should we bump up this capacity?

Reply:
Actually I don't really know what is this property about at all :) probably it's a question to @littleaj that have bumped maximum value from 100 to 1000 recently.

Reply:
@gavlyukovskiy  this property is used for disk persistence. When the network fails the telemetry gets stored in the disk and when it is restored we transmit that telemetry to the cloud.

Reply:
@dhaval24 nice to know, added your explanation to the documentation :)